psf-new-code-of-conduct.md

```diff
@@ -44,16 +47,16 @@ Examples of unacceptable behavior by participants include:
 
 Community members asked to stop any inappropriate behavior are expected to comply immediately.
 
-### Reversisms
+### Communicating About Assumptions and Boundaries
 
-The Python community prioritizes marginalized people's safety over privileged people's comfort. The Python Code of Conduct team will not act on complaints regarding:
+The Python Community welcomes people who have traditionally been marginalized within our community: we allow them the space to discuss behaviors, stereotypes, and assumptions that make them feel unwelcome. We also believe that helping others understand how various communications are received can be constructive.
```

Comment:
```suggestion
The Python Community welcomes people who have traditionally been marginalized within our community: we allow them the space to discuss behaviours, stereotypes, and assumptions that make them feel unwelcome. We also believe that helping others understand how various communications are received can be constructive.
```
Just normalizing the spelling. 😄

Reply:
@brettcannon Thanks for your input; I expect to standardise on en-US, unfortunately.

Reply:
I figured, but I at least enjoyed the thought of the alternative spelling, even if for a brief moment. 😄